### PR TITLE
Add file .gitignore

### PR DESCRIPTION
**Pull request by hsyuan**

### CODE REVIEW

**Comment by hsyuan** [comment]
@vraghavan78 @xinzweb @oarap 
Can you please take a look?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+*.mo
+*.html
+*.css
+*.targz
+.obj.*
+
+# Local excludes in root directory
+build
+server/minidumps/
+ivy-*.xml
```
Comment (vraghavan78) [inline .gitignore]: Can you add:

.autotools
.cproject
.gitignore
.project

**Comment by vraghavan78** [comment]
LGTM Thanks @hsyuan

**Comment by oarap** [comment]
LGTM too

File: .gitignore (hunk 1 above)
Comment (hsyuan) [inline .gitignore]: Added folders mentioned above except .gitignore. 
.gitignore should be tracked.

**Comment by hsyuan** [comment]
Merged to master https://github.com/greenplum-db/gporca/commit/1a04a5ce6a0f3adcb562943be5f8ab964c9e10b1

**Pull request closed without merging**